pom.xml

```diff
@@ -209,7 +209,7 @@
       <dependency>
         <groupId>org.assertj</groupId>
         <artifactId>assertj-core</artifactId>
-        <version>1.7.1</version>
+        <version>3.0.0</version>
```

Comment:
The new version of assertj has richer matchers including for java 8 types. In this case, I update to assert on Path instances.